fmc/src/flow/pcr.rs

```diff
@@ -23,9 +23,30 @@ Note:
 use crate::flow::tci::Tci;
 use crate::fmc_env::FmcEnv;
 use crate::HandOff;
-use caliptra_drivers::{okref, CaliptraResult, PcrId};
+use caliptra_drivers::{okref, CaliptraError, CaliptraResult, PcrBank, PcrId, Sha384};
 
-use caliptra_common::{RT_FW_CURRENT_PCR, RT_FW_JOURNEY_PCR};
+use caliptra_common::{
+    memory_layout::{PCR_LOG_ORG, PCR_LOG_SIZE},
+    PcrLogEntry, PcrLogEntryId, RT_FW_CURRENT_PCR, RT_FW_JOURNEY_PCR,
+};
+use zerocopy::AsBytes;
+
+struct PcrExtender<'a> {
+    pcr_bank: &'a mut PcrBank,
+    sha384: &'a mut Sha384,
+}
+impl PcrExtender<'_> {
+    fn extend(
+        &mut self,
+        pcr_id: PcrId,
+        bytes: &[u8],
+        pcr_entry_id: PcrLogEntryId,
+    ) -> CaliptraResult<()> {
+        self.pcr_bank
+            .extend_pcr(PcrId::PcrId0, self.sha384, bytes)?;
+        log_pcr(pcr_entry_id, pcr_id, bytes)
```

Comment:
With https://github.com/chipsalliance/caliptra-sw/pull/600, we could adjust this logic a bit such that we only create one log when extending into both PCRs. (This wouldn't be strictly necessary, we could keep creating one log for each extend operation, but it's more space efficient if we can group them together).

Reply:
Can you make that code common?